recipe/meta.yaml

```diff
@@ -38,6 +37,11 @@ requirements:
     - pytorch >=1.8.0
     - tqdm >=4.56.0
     - setuptools <=59.5.0
+    - jax >=0.3
```

Comment:
```suggestion
    - jax >=0.3
    - jaxlib <0.3
```

Reply:
We have fixed the `jaxlib` issue and you should be able to revert this line.